src/scripts/main.js

```diff
@@ -1,3 +1,129 @@
 'use strict';
 
-// write your code here
+function keyListener(e) {
+  if (game.getStatus() !== 'playing') {
+    return;
+  }
+
+  switch (e.key) {
+    case 'ArrowDown':
+      game.moveDown();
+      break;
+    case 'ArrowLeft':
+      game.moveLeft();
+      break;
+    case 'ArrowRight':
+      game.moveRight();
+      break;
+    case 'ArrowUp':
+      game.moveUp();
+      break;
+  }
+
+  update();
+}
+
+function update() {
+  game.getState().forEach((row, i) => {
+    row.forEach((el, j) => {
+      let cellCssClass = 'field-cell';
+
+      cellCssClass += el >= 2 ? ` field-cell--${el}` : '';
+      gameCells[i][j].innerText = el || '';
+      gameCells[i][j].className = cellCssClass;
+    });
+  });
+
+  scoreElement.innerText = game.getScore();
+
+  switch (game.getStatus().toUpperCase()) {
```

Comment:
The 'toUpperCase' method is used to normalize the game status, but this could be error-prone if the status values change. Consider using constants for the status values to prevent such issues.